docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,151 @@
+:::
+
+To avoid any issues with a node's private key you must specify an alternate location for the **tentacle.config** file upon installation of the Tentacle service. Follow these instructions to install Octopus Tentacle in a failover cluster using shared storage.
+
+## Requirements {#ClusteringTentacles-Requirements}
+
+This document assumes you already have the following setup:
+
+- An Active Directory Domain and a local DNS server.
+- An Octopus Server, this does not need to be joined to the domain.
+- A two node active/passive windows cluster where each node is joined to the domain.
+- A local IP Address available for the cluster so that the Octopus Server can reach the cluster.
+- Shared Storage configured for the cluster, this example has mounted it as E:\
+
+## Installation {#ClusteringTentacles-Installation}
+
+:::warning
+**Installing Tentacles with Shared storage**
+This document implements shared storage using an iSCSI target with Multipath IO confgured on the Tentacle servers, in this scenario it is best to avoid accessing the same iSCSI volume from two different hosts at the same time.
+:::
+
+On the first node in your Active/Passive Windows cluster, check that the shared drive is mounted by seeing that it is available in disk management and note down the Drive letter. If it hasn't already been done, bring the shared disk online and format it with a drive letter, we have chosen to use E:\
+
+*![](/docs/images/clustered-listening-tentacles/shared-disk-properties.jpg)*
+
+Run through the Tentacle MSI Installer to install Tentacle Manager to its default location (C:\Program Files\Octopus Deploy\Tentacle). Do not click "get started" in the Tentacle manager, install the Octopus Tentacle instance using the command prompt by opening `cmd` then run these commands:
+
+```
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe create-instance --instance "Tentacle" --config "C:\Octopus\Tentacle.config"
+Tentacle.exe new-certificate --instance "Tentacle" --if-blank
+Tentacle.exe configure --instance "Tentacle" --reset-trust
+Tentacle.exe configure --instance "Tentacle" --app "E:\Octopus\Applications" --home "E:\Octopus\Home" --port "10933" --noListen "False"
+Tentacle.exe configure --instance "Tentacle" --trust "<OCTOPUS SERVER THUMBPRINT>"
+"netsh" advfirewall firewall add rule "name=Octopus Deploy Tentacle" dir=in action=allow protocol=TCP localport=10933
+Tentacle.exe service --instance "Tentacle" --install --stop --start
+```
+In the script block above, the following actions were performed:
+ - Installed the tentacle instance using the default instance name of "Tentacle" and make sure the Tentacle.config file was installed into the default location of "C:\Octopus\Tentacle.config". Since C:\ is local storage this file does not get swapped when we perform a swap to a new active node.
```

Comment:
Mixing present and past tense. `make` => `made`.

Backticks around paths and drive letters